.github/workflows/test.yml

```diff
@@ -35,3 +35,6 @@ jobs:
     - run: npm test
     - run: node ./bin.js trace test
     - run: node ./bin.js "**/{*.ts,*.md}"
+
+    # Ensure the repository is clean after build & test
+    - run: git --no-pager diff --compact-summary --exit-code
```

Comment:
Good idea 👍

Reply:
For now, I'm adding it to the build so I can see the differences.
But since there is a binary difference, I need to investigate why.

Reply:
The binary difference is OK I think, it's just letting you know that you didn't check in the latest built version of the dictionary